content/webapp/services/prismic/transformers/exhibition-guides.ts

```diff
@@ -117,43 +124,36 @@ export function transformExhibitionGuide(
   const { data } = document;
 
   const components: ExhibitionGuideComponent[] = data.components?.map(
-    component => {
+    (component: ExhibitionGuideComponentPrismicDocument) => {
```

Comment:
This line, and possibly the changes to `ExhibitionGuidePrismicDocument`, are really key.

Without them, the type checker couldn't get a more specific type for `component` than `any`. That means all the downstream types are `any`, and it can't do any type checking on this whole block.